### PR TITLE
add initial scene

### PR DESCRIPTION
**Pull request by ochanje210**

### CODE REVIEW

File: shared/components/initialScene.js [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+import {Image, Platform, View, Text, Dimensions, TouchableOpacity} from 'react-native';
+import Swiper from 'react-native-swiper';
+import { Actions } from 'react-native-router-flux';
+
+import ImgLogin1 from '../resources/initialScene/loginImage1.png';
+import ImgLogin2 from '../resources/initialScene/loginImage2.png';
+import ImgLogin3 from '../resources/initialScene/loginImage3.png';
+
+const WindowHeight = 477.8 + 162;
+const WindowWidth = 360;
```
Comment (yuda1124) [inline shared/components/initialScene.js]: It depends on devices. isn't it?

File: shared/components/initialScene.js [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+    this.props.setCurrentScene('map');
+    Actions.map({type: 'replace'});
+  }
+  render() {
+    return (
+      <View style={{flexDirection: 'column', flex: 1}}>
+        <View style={{flex: 1}}>
+          {this.renderSwiper()}
+        </View>
+        <View style={[styles.bottomBox, {height: Dimensions.get('window').height * BottomBoxHeight / (SwiperHeight + BottomBoxHeight)}]}>
```
Comment (heartjigi) [inline shared/components/initialScene.js]: It's to long, It cause lint error

File: shared/components/initialScene.js [hunk 3]
```diff
@@ -0,0 +1,184 @@
+import React, {Component, PropTypes} from 'react';
+import {Image, Platform, View, Text, Dimensions, TouchableOpacity} from 'react-native';
+import Swiper from 'react-native-swiper';
+import { Actions } from 'react-native-router-flux';
+
```
Comment (ace0625) [inline shared/components/initialScene.js]: space?

File: shared/components/initialScene.js [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+import ImgLogin1 from '../resources/initialScene/loginImage1.png';
+import ImgLogin2 from '../resources/initialScene/loginImage2.png';
+import ImgLogin3 from '../resources/initialScene/loginImage3.png';
+
+const WindowHeight = 477.8 + 162;
```
Comment (ace0625) [inline shared/components/initialScene.js]: Why static window size? any reason?

File: shared/components/initialScene.js (hunk 1 above)
Comment (ochanje210) [inline shared/components/initialScene.js]: yes, these values are just reference to some ratio we had from the design.

Comment (ochanje210) [inline shared/components/initialScene.js]: so, these values won't directly applied to the component's styles as you will see below

File: shared/components/initialScene.js (hunk 2 above)
Comment (yuda1124) [inline shared/components/initialScene.js]: I already delete that in eslintrc.

File: shared/components/initialScene.js (hunk 3 above)
Comment (ochanje210) [inline shared/components/initialScene.js]: this does not trigger eslint warning.
also, this is intended to separate importing library or importing images and others

**Comment by yuda1124** [comment]
good work! LGTM

File: shared/components/initialScene.js [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+      }
+    })
+  }
+};
+
+class InitialScene extends Component {
+  constructor(props) {
+    super(props);
+  }
+  renderSlide(style, upperText, belowText, image) {
```
Comment (ace0625) [inline shared/components/initialScene.js]: need an empty line between functions.

**Pull request merged**